install_notifier.js

```diff
@@ -0,0 +1,7 @@
+// Dispatch a message to every URL that's in the manifest to say that RECAP is
+// installed.
+let currentVersion = chrome.runtime.getManifest().version;
```

Comment:
This makes me uncomfortable, but I'm not quite sure if I'm right — I've still not entirely wrapped my head around `let`. But as I understand it, `let` at the top of a file is not guaranteed to be file scoped. https://esdiscuss.org/topic/let-and-file-scope suggests that top-level `let`s in a file may be global-scoped, unless this is one of the exceptions.

So I'd feel more comfortable if this was wrapped in a closure or a block.
(Also, I'm not sure you need a variable there. You could just move the `getManifest()` call down 3 lines.)

Reply:
Tweaked to be a closure. Good call.